srcpkgs/nautilus/template

```diff
@@ -16,8 +16,8 @@ short_desc="GNOME file manager"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
-distfiles="${GNOME_SITE}/${pkgname}/${version::4}/${pkgname}-${version}.tar.xz"
-checksum=f9252f508211d9f368a49db697890f895d080f09590aecaf33803a519def7bde
+distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
+checksum=b6cafc7ab1e70a64383de391b6097fcccbf36b208f8502d8c46423224fd30ef8
```

Comment:
`move-files` and `copy-files` tests are failing, looks serious.
```diff
 srcpkgs/nautilus/template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/srcpkgs/nautilus/template b/srcpkgs/nautilus/template
index 94cb10492c..c6faf7bf3d 100644
--- a/srcpkgs/nautilus/template
+++ b/srcpkgs/nautilus/template
@@ -11,7 +11,7 @@ makedepends="exempi-devel gnome-autoar-devel gnome-desktop-devel gtk+3-devel
  libX11-devel libxml2-devel tracker-devel gst-plugins-base1-devel"
 depends="desktop-file-utils gsettings-desktop-schemas hicolor-icon-theme
  tracker-miners tracker"
-checkdepends="tracker"
+checkdepends="tracker tracker-miners"
 short_desc="GNOME file manager"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
```

Reply:
@sgn Added.